### ISSUE TITLE
PARSE ERROR with upload successfull

### ISSUE THREAD

**Issue by belsanpesou** [fengyuanchen/cropper#535]
Hi Feng,

I have a problem intermittent.It is that even with uploading been successfully done, the message of parseError appears and the window does not close. The closing is disabled and  thumbnail generation function not work. And the worst is that even avoiding the error messages, the operation is not completed, but the operation has been sucessfull . Can you help me?

**Comment by fengyuanchen** [comment]
Do you read the [README.md](https://github.com/fengyuanchen/cropper/blob/master/examples/crop-avatar/README.md) about the [Crop Avatar](https://github.com/fengyuanchen/cropper/blob/master/examples/crop-avatar) example?

**Comment by belsanpesou** [comment]
Yes, I read. But the  parse error descripted on  Network say nothing. The example  not works too. 
I tested on three different servers and the cropper did not work properly. The reason is that the jQuery ajax was not getting the answer correctly. In this case, the jQuery response only worked when put in the share target file - crop.php the 'date_default_timezone_set' function of php concealing errors. That is, if the php_ini is configured to display any errors, the jQuery ajax probably will indicate parseError.

**Comment by fengyuanchen** [comment]
The crop.php should output pure JSON only.

**Comment by belsanpesou** [comment]
I know. I managed to solve this problem. JQuery would only take off the crop class that returns JSON data. Only if php is configured to display all errors, alerts appears along with the json output. so the jQuery indicated error: he took not only the json data as specified in the ajax call, but the full output of crop.php file - which contained php alert strings. To end the parse error, I had to stop all php output alerts, which in my case is to place at the beginning the date_default_timezone_set function (could work with error_reporting (0));

![sem titulo-1](https://cloud.githubusercontent.com/assets/4975784/11659978/0485fb5c-9db2-11e5-93d2-aa77f628ef52.png)
